default/scripting/species/SP_SLY.focs.txt

```diff
@@ -97,19 +101,7 @@ Species
             effects = UnlockPolicy name = "PLC_CONFEDERATION" empire = Source.Owner
     ]
 
-    environments = [
-      type = Swamp        environment = Hostile
-      type = Toxic        environment = Hostile
-      type = Inferno      environment = Hostile
-      type = Radiated     environment = Hostile
-      type = Barren       environment = Hostile
-      type = Tundra       environment = Hostile
-      type = Desert       environment = Hostile
-      type = Terran       environment = Hostile
-      type = Ocean        environment = Hostile
-      type = Asteroids    environment = Uninhabitable
-      type = GasGiant     environment = Good
-    ]
+    [[GASEOUS_STANDARD_EP]]
```

Comment:
there definitely need to be more native species for roids and giants, because both are most common type of planet and during natives placement phase of universe creation it doesnt care about planet type frequencies, just picks random planets and places a random species from all species that can live there. so, statistically, there are many more roids and giants but only one liveable species each, so they get many more homeworlds relative to other natives. was already discussed on the forum, .. where was i. short answer: yes. or: right you are.

Reply:
certainly also could change picking planets or, if the chosen species was chosen very often, choose a completely random species and transmute the planet to the one of the best environments of that species